README.md

```diff
@@ -21,13 +16,26 @@ To install `mesheryctl`, execute the following commands.
 ```
 brew tap layer5io/tap
```

Comment:
Let's change it to `meshery/tap` now? cc: @leecalcote

Reply:
@nishantwrp is the installation command changed ??

Reply:
Yup, when we do a `brew tap <org>/<repo>`. Homebrew looks for a github repo with url: `https://www.github.com/<org>/homebrew-<repo>`. Since this repo was moved from `layer5io` org to `meshery` org. Ideally, we should change the command too.

Reply:
okay, so do we need to update the readme with the latest command:
 `brew tap meshery/tap`

Reply:
@nishantwrp thank you for this diligence. Sound good to me, @manikmmalhotra.

Reply:
@leecalcote @nishantwrp updated the readme also i'll make a PR for documentation
![Screenshot from 2022-07-09 19-17-37](https://user-images.githubusercontent.com/63007399/178108610-27c27b23-755b-4e41-9bcf-836c9c474588.png)

Reply:
@manikmmalhotra, I'll let @nishantwrp offer review here.